libvast/src/system/partition_transformer.cpp

```diff
@@ -378,7 +378,8 @@ partition_transformer_actor::behavior_type partition_transformer(
         // Create the value indices.
         for (auto& [qf, idx] :
              self->state.partition_buildup.at(data.id).indexers)
-          data.indexer_chunks.emplace_back(qf.name(), chunkify(idx));
+          data.indexer_chunks.emplace_back(qf.name(),
+                                           idx ? chunkify(idx) : nullptr);
       }
```

Comment:
If we encounter an indexer chunk that is empty, should we also add special handling to just treat it as if it were a nullptr just in case users ran into the bug?

As-is, this would only fix the issue for newly rebuilt partitions. Were we already able to run into this with v2.2?

Reply:
It only became possible to trigger the bug with #2509 so v2.2 should be fine, but I think it makes sense anyways as a defensive coding measure.

Reply:
Okay, but then at least we don't need a changelog entry.

Reply:
NB: You pushed an empty changelog file that should probably be removed again. Other than that this is ready to go imo.

Reply:
@lava I unresolved to get you to see this comment

> NB: You pushed an empty changelog file that should probably be removed again. Other than that this is ready to go imo.

Reply:
I'll resolve again to avoid re-running CI, but I'll fix it in the rc3 PR.

Reply:
Okay that's fine by me, just wanted to make sure you noticed.